ui/pnpm-lock.yaml

```diff
@@ -41,7 +41,7 @@ importers:
       '@tailwindcss/forms': 0.5.0_tailwindcss@3.0.23
       '@testing-library/dom': 8.11.3
       '@testing-library/svelte': 3.1.0_svelte@3.47.0
-      '@testing-library/user-event': 13.5.0_gzufz4q333be4gqfrvipwvqt6a
+      '@testing-library/user-event': 13.5.0_@testing-library+dom@8.11.3
```

Comment:
Why do we have all these changes in `ui/pnpm-lock.yaml`?

Reply:
fixed